src/api/Feed.js

```diff
@@ -505,9 +507,12 @@ class Feed extends Base {
 
         this.updateFeedItem({ isRepliesLoading: true }, commentFeedItemId);
 
-        const successCallbackFn = (comments: ThreadedCommentsType) => {
-            this.updateFeedItem({ isRepliesLoading: false, replies: comments.entries }, commentFeedItemId);
-            successCallback(comments.entries);
+        const successCallbackFn = ({ entries }: ThreadedCommentsType) => {
```

Comment:
If this is only using the entries, should we pass in entries field instead of the object?

Reply:
In this case we're using only entries but I think the API should return the whole response including marker data as this is the common behaviour in all API methods.